static/js/app.ts

```diff
@@ -1027,6 +1027,17 @@ export function toggle_developers_mode(example_programs: boolean) {
   }
 }
 
+export function load_profile(username: string, mail: string, birth_year: number, gender: string, country: string) {
+  $('#profile').toggle();
+  if ($('#profile').is(":visible")) {
```

Comment:
Zou de gekozen taal dan hier niet ook bij moeten staan?

Reply:
Als ik het me goed herinner halen we die uit de session en zetten hem dan in de form

Reply:
Deze functie wordt alleen gebruikt voor het handig parsen van de session values naar het html form